app/views/groups/_groups_manager.js.jsx.erb

```diff
@@ -280,7 +280,11 @@
       var groups_data = this.props.groups.map(function(group) {
         var g = {};
         g['id'] = group.id;
+        <% if @assignment.scanned_exam %>
+        g['name'] = <a href={'groups/assign_scans?group=' + group.id}>{group.name}</a>;
```

Comment:
does a path helper still work if I need to input javascript variables like `group.id` and `group.name`?